lib/metrics.js

```diff
@@ -28,6 +28,8 @@ const ERROR_METRIC_NAMES = [
 
 const LIB_RELEASE_PATH = '/etc/os-release';
 const LIB_RELEASE_PATH_SECONDARY = '/usr/lib/os-release';
+const CONTAINER_MEMORY_SIZE_V1 = '/sys/fs/cgroup/memory/memory.limit_in_bytes';
```

Comment:
nitpick: would it be better to prefix the version with CGROUP so that we know what version it is?